lib/modules/manager/ansible-galaxy/extract.ts

```diff
@@ -87,7 +87,7 @@ export function extractPackageFile(
     }
     return { deps };
   } catch (err) /* istanbul ignore next */ {
-    logger.debug({ err }, 'Error extracting ansible-galaxy deps');
+    logger.debug({ err, packageFile }, 'Error extracting ansible-galaxy deps');
```

Comment:
@rarkins @secustor @JamieMagee Should we warn instead to make those errors more visible on dashboard? As those errors are mostly caused by wrong config or invalid files.

Reply:
There's some cases where it's probably worthwhile, but others where it's probably just noise. No single answer

Reply:
so I would prefer to always only debug log as I've done it here